src/handlers/autolabel.rs

```diff
@@ -12,6 +12,27 @@ pub(super) fn parse_input(
     event: &IssuesEvent,
     config: Option<&AutolabelConfig>,
 ) -> Result<Option<AutolabelInput>, String> {
+    if event.action == IssuesAction::Opened {
```

Comment:
We probably also want to do this on synchronize(?) so that we finally fix the bugs with files pushed after first opening not being detected.

I suspect it's potentially as simple as piping through that issue action, so maybe worth doing?